src/components/Filters/FilterModal.tsx

```diff
@@ -50,22 +48,27 @@ const defaultFilterCompareFn: FilterFieldCompareFn = (a, b) => {
 };
 
 const FilterInput = (props: FilterInputProps) => {
+	const { schema, value, onUpdate } = props;
```

Comment:
I guess we can inline the destructuring and stop using `props.X` completely :)
```suggestion
const FilterInput = ({ schema, value, onUpdate }: FilterInputProps) => {
```